### PR TITLE
Industrial Edge copy images utility

### PR DESCRIPTION
**Pull request by claudiol**
This script uses skopeo to copy the industrial edge boostrap images for iot-consumer iot-anomaly-detection iot-frontend iot-software-sensor to the image registry that is defined in the values-global.yaml file.

### CODE REVIEW

File: scripts/industrial-edge-image-copy.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#   \033[0K\r - Trailing escape sequence to leave output on the same line
+function log {
+    if [ -z "$2" ]; then
+        echo -e "\033[0K\r\033[1;36m$1\033[0m"
+    else
+        echo -e $1 "\033[0K\r\033[1;36m$2\033[0m"
+    fi
+}
+
+function parse_yaml {
```
Comment (mbaldessari) [inline scripts/industrial-edge-image-copy.sh:18]: Maybe asking for yq to be installed and fetch the few keys we need is simpler?

Comment (claudiol) [reply scripts/industrial-edge-image-copy.sh:18]: Agreed ... let's refactor afterwards

File: scripts/industrial-edge-image-copy.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+# User needs to login to the registry
+registryLogin
+
+if [ -f $LOGFILE ]; then
+  rm -f $LOGFILE
+fi
+touch $LOGFILE
+
+for image in iot-consumer iot-anomaly-detection iot-frontend iot-software-sensor
```
[the remark this answers is not in the document]
Comment (claudiol) [reply scripts/industrial-edge-image-copy.sh:88]: Ack.  I can add this as well.

[the remark this answers is not in the document]
Comment (claudiol) [reply scripts/industrial-edge-image-copy.sh:86]: Done

File: scripts/industrial-edge-image-copy.sh [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    continue
+  else
+    if [ -d $location ]; then
+      break;
+    else
+      log -n "Please enter location of values-global.yaml: LOCATION MUST BE A DIRECTORY. Press Enter to continue"
+      read dummy
+      continue
+    fi
+  fi
```
[the remark this answers is not in the document]
Comment (claudiol) [reply scripts/industrial-edge-image-copy.sh:73]: Ok cool ... let me update this as well.

**Pull request merged**